packages/messaging/lib/index.js

```diff
@@ -448,6 +479,18 @@ class FirebaseMessagingModule extends FirebaseModule {
     this._isDeliveryMetricsExportToBigQueryEnabled = enabled;
     return this.native.setDeliveryMetricsExportToBigQuery(enabled);
   }
+
+  async isSupported() {
+    if (Platform.isAndroid) {
+      return this.native.isSupported();
```

Comment:
🤔  we are guaranteed to have the app module loaded because it's a strict requirement for react-native-firebase, and it always loads the native module named `RNFBUtilsModule` which exports a constant here we could reference similar to how the Utils statics off app do it, meaning we could do away with the native implementation here and delegate to the existing on already here https://github.com/invertase/react-native-firebase/blob/94536b024b4db404ba7c6bb7f3735291a4a4642e/packages/app/android/src/reactnative/java/io/invertase/firebase/utils/ReactNativeFirebaseUtilsModule.java#L145

The status map has an `isAvailable` boolean on it, so I think this API could delegate like so?

https://github.com/invertase/react-native-firebase/blob/94536b024b4db404ba7c6bb7f3735291a4a4642e/packages/app/e2e/utils.e2e.js#L35-L37

```suggestion
      playServicesAvailability = firebase.utils().playServicesAvailability;
      return playServicesAvailability.isAvailable;
```

I *think* that will work anyway? There may be some sort of javascript import junk to handle, or perhaps a better way to gain access to the symbol since we should have internal access to it (that is: a different way to import and use it vs `firebase.utils().NNN`) ?

Sorry for beating this one to death - what I'm hoping for is that in the end it really is "that easy" - the native implementation is already done in app/utils so I was thinking it was just an easy reference to it